bson-kotlin/src/main/kotlin/org/bson/codecs/kotlin/DataClassCodec.kt

```diff
@@ -85,6 +85,8 @@ internal data class DataClassCodec<T : Any>(
                 if (logger.isTraceEnabled) {
                     logger.trace("Found property not present in the DataClass: $fieldName")
                 }
+            } else if (propertyModel.param.type.isMarkedNullable && reader.currentBsonType == BsonType.NULL) {
+                reader.skipValue()
```

Comment:
Optional: `reader.readNull()` is a bit clearer in intent, and a bit more efficient as well.

Reply:
Ack